docs/source/cloud/architecture.rst

```diff
@@ -0,0 +1,66 @@
+===========================================
+The Registry API Service Cloud Architecture
+===========================================
+
+On-Prem vs. Cloud
+*****************
+
+In a typical on-premise deployment, the PDS Registry Application is composed, server-side of
```

Comment:
I feel it is better to reword this (from line 8 to 11).

Reply:
I tried to simplify the sentence, these are not my words so I am not necessarily comfortable in doing that but I agree this sentence is a bit complicated, although it sounds like good english.